src/components/HeroesItem/HeroesItem.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link } from 'react-router-dom'
+import { Hero } from "../../types/hero";
+import './heroesItem.scss'
+
+interface IProps {
+  hero: Hero
+}
+export const HeroesItem = (props: IProps) => {
+  const { hero } = props
+
+
+  return (
+    <div className="heroes">
+
+      <div className="heroes__avatar">
+          <img src={`${hero.thumbnail.path}.${hero.thumbnail.extension}`} alt="avatar" />
```

Comment:
better to create separate variable for image src

Reply:
done